components/groups/list_item_groups/NoActionListItemGroup.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card
+    class="mx-auto"
+    max-width="400"
+  >
+    <v-list flat>
```

Comment:
It should be
```
<v-list-group> 
  ... 
   <v-list-group> 
      ...
   </v-list-group>
</v-list-group>
```

For example:
```suggestion
<template>
  <v-card class="mx-auto" max-width="400">
    <v-list>
      <v-list-group :value="true" no-action>
        <template v-slot:activator>
          <v-list-item-title>Test</v-list-item-title>
        </template>
        <v-list-group :value="true" color="indigo" no-action sub-group>
          <template v-slot:activator>
            <v-list-item-content>Test2</v-list-item-content>
          </template>
          <v-list-item v-for="(item, i) in items" :key="i" link>
            <v-list-item-icon>
              <v-icon v-text="item.icon"></v-icon>
            </v-list-item-icon>
            <v-list-item-content>
              <v-list-item-title v-text="item.text"></v-list-item-title>
            </v-list-item-content>
          </v-list-item>
        </v-list-group>
      </v-list-group>
    </v-list>
  </v-card>
</template>```

Reply:
Done. But there is an issue. If the list-item-group is closed once, it's not opening again

Reply:
I suppose you have a error in implementation
I've checked with my example and it works correct

Reply:
Fixed